project-addons/purchase_picking/models/stock.py

```diff
@@ -160,7 +160,9 @@ def write(self, vals):
                     and 'origin' not in vals\
                     and not move._context.get('accept_ready_qty'):
                 if move.product_uom_qty > vals['product_uom_qty'] > 0:
-                    move.copy({'picking_id': False, 'product_uom_qty': move.product_uom_qty - vals['product_uom_qty']})
+                    if not (move.location_id.name == "Vendor's deposit"
```

Comment:
Esta ubicación está creada en código, podéis cogerla con self.env.ref("stock_location_vendor_deposit") y no dependeis de un campo de texto

Reply:
Pero está creado solo en Italia, porque viene del módulo automatize_edi_it. Entiendo que en españa daría error al no encontrar la referencia externa

Reply:
Sí, ok no me fijara en el módulo